service/src/test/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelperTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class EmailNotificationHelperTest {
+
+    @Test
+    void subjectLineShouldBeEmptyWhenNoRespondentOrCaseNumberEmpty() {
```

Comment:
Do need to a test to ensure that if respondent[0] has no name but respondents[1] has a name, it will return back empty string?

Reply:
added